docs/en/core-concepts.md

```diff
@@ -0,0 +1,10 @@
+# Core concepts
```

Comment:
It's a trivial thing but camelizing all words is better like other chapter title.
`Core Concepts`